Problem0.cs

```diff
@@ -0,0 +1,44 @@
+﻿namespace ProtoHackers;
+using System.IO.Pipelines;
+using System.Net.Sockets;
+using System.Net;
+
+public class Problem0_EchoServer
+{
+    public static async Task Init(int port)
+    {
+        var listenSocket = new Socket(SocketType.Stream, ProtocolType.Tcp);
+        listenSocket.Bind(new IPEndPoint(IPAddress.Any, port));
+
+        listenSocket.Listen();
+
+        while (true)
+        {
+            var socket = await listenSocket.AcceptAsync();
+            _ = Handle(socket);
```

Comment:
I would `Task.Run` Handle instead of just ignoring the task.

Reply:
Not sure I understand here. It could be
```
_ = Task.Run(() => Handle(socket))
```
but isn't that still ignoring the task? 

I do see the advantage of `Task.Run` because, if I understand correctly, it would dispatch to thread pool immediately instead of potentially blocking the main socket accept loop until the first yield in `Handle`.